.github/CODEOWNERS

```diff
@@ -19,3 +19,6 @@ x-pack/plugin/monitoring/src/main/java/org/elasticsearch/xpack/monitoring/Monito
 # Fleet
 x-pack/plugin/fleet/src/main/java/org/elasticsearch/xpack/fleet @elastic/fleet
 x-pack/plugin/core/src/main/resources/fleet-* @elastic/fleet
+
+# Kibana Security
+x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/authz/store/KibanaOwnedReservedRoleDescriptors.java @elastic/kibana-security
```

Comment:
__question:__ Does it make sense to add your team as a co-owner as well? I suppose currently you are only mentioned if specific labels are added. I'm not sure if you want to change that, though.

Reply:
Good q! I think we should stick with the existing process for now, since it's already established and working. We can always re-assess later.